### PR TITLE
Update AWS documentation on ASGs

### PR DESCRIPTION
**Pull request by DavidWittman**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1730)

### CODE REVIEW

File: vendor/k8s.io/kubernetes/docs/design/aws_under_the_hood.md [hunk 1]
```diff
@@ -76,15 +76,12 @@ multiple ephemeral instance disks.
 
 ### Auto Scaling group
 
-Nodes (but not the master) are run in an
-[Auto Scaling group](http://docs.aws.amazon.com/AutoScaling/latest/DeveloperGuide/AutoScalingGroup.html)
-on AWS. Currently auto-scaling (e.g. based on CPU) is not actually enabled
+Two 
+[Auto Scaling groups](http://docs.aws.amazon.com/AutoScaling/latest/DeveloperGuide/AutoScalingGroup.html) are created
+on AWS: one for the masters and one for the nodes. Currently auto-scaling (e.g. based on CPU) is not actually enabled
```
Comment (zytek) [inline vendor/k8s.io/kubernetes/docs/design/aws_under_the_hood.md]: One ASG per master per zone

Comment (DavidWittman) [inline vendor/k8s.io/kubernetes/docs/design/aws_under_the_hood.md]: > [Auto Scaling groups](http://docs.aws.amazon.com/AutoScaling/latest/DeveloperGuide/AutoScalingGroup.html) are created for the masters and nodes. One group is created for the nodes and one group is created per master per AZ.

Better?

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

File: vendor/k8s.io/kubernetes/docs/design/aws_under_the_hood.md (hunk 1 above)
Comment (zytek) [inline vendor/k8s.io/kubernetes/docs/design/aws_under_the_hood.md]: yes :)

**Comment by justinsb** [comment]
Thanks @DavidWittman !  But... this is in `vendor`, i.e. it isn't really part of kops.  The primary document is here now:  https://github.com/kubernetes/community/blob/master/contributors/design-proposals/aws_under_the_hood.md

We also should draw a distinction between what kube-up and what kops does.  The original document doesn't make that distinction.

If you send a similar PR upstream, do cc me.

Also, if you want to create a similar PR here describing the autoscaling groups etc (in docs), that would be great.  It doesn't need to be perfect either, I find that having a starting point is the hardest thing!

OK if I close this, because we can't merge this?

**Comment by DavidWittman** [comment]
Whoops... I didn't even realize these docs were vendored. I'll submit a new
PR upstream. Thanks!

**Comment by justinsb** [comment]
Thanks @DavidWittman - do cc me for review!  I'll close this one :-)

**Pull request closed without merging**